dp-cluster-setup-utility.py

```diff
@@ -1186,6 +1538,57 @@ def knox_ip(self):
       return socket.gethostbyname(self.knox_host)
     except Exception:
       return None
+class CMPrerequisites:
+  def __init__(self, cm):
+    self.cm = cm
+    self.knox_host = cm.cluster.knox_host()
+
+  def satisfied(self):
+    if not self.stack_supported():
+      print('The stack version (%s) is not supported. Supported stacks are: CDH-6.0 or newer.' % self.cm.installed_stack())
+      return False
+    if not self.security_type_supported():
+      print('Your cluster is not kerberied. Please enable Kerberos using Ambari first.')
```

Comment:
I would suggest we pass this for now. We will get to it as part of knox support.